scss/jetpack-deactivate-dialog.scss

```
@@ -0,0 +1,174 @@
+#TB_window.jetpack-disconnect-modal {
+
+	max-width: 635px;
+	max-height: 600px;
+
+	@media screen and (min-width: 661px) {
+		max-height: 360px;
+	}
+
+	#TB_title {
+		font-size: 28px;
+		line-height: 24px;
+		font-weight: 600;
+		height: 24px;
+		padding: 24px;
+		color: #23282d;
```

I would recommend avoiding hardcoding those colors, and instead rely on variables defined elsewhere when possible. I've added a few examples below, and I would encourage you to review the whole file for values you can use.

```suggestion
		color: $wpui-header-dark;
```